apiserver/pkg/model/converter.go

```diff
@@ -14,6 +15,45 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+func getNodeDefaultAnnotations() []string {
```

Comment:
nit: In the future, we can list the constant string below separately, like:https://github.com/ray-project/kuberay/blob/25864689ba31214bfe3469b762d2dac29ce72f89/apiserver/pkg/util/config.go#L11-L18

Reply:
Done